frontend/src/pages/Editor/Editor.tsx

```diff
@@ -101,7 +120,7 @@ const Editor = () => {
       <ShortcutGuide />
 
       <ExportImage />
-
+m
```

Comment:
![images(6).jpg](https://github.com/automatarium/automatarium/assets/19339842/176bf069-73c4-4dea-bc9d-8c7db8926215)

Reply:
but there do be an "m" at the bottom of the page tho
![image](https://github.com/automatarium/automatarium/assets/19339842/8a3c14ca-b9e5-426c-a1d8-8396467df3bd)